### PR TITLE
Slovenia lang

### PR DESCRIPTION
**Pull request by extra90**
Adds initial slovenian language pack.

### CODE REVIEW

File: sl/pagination.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	| the simple pagination links. You are free to change them to anything
+	| you want to customize your views to better match your application.
+	|
+	*/
+
+	'previous' => '&laquo; Prejšnja',
+
+	'next'     => 'Naslednja &raquo;',
+
+    'first'    => 'Prvi',
```
Comment (caouecs) [inline sl/pagination.php]: Hello,
no need

File: sl/pagination.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    'last'     => 'Zadnji',
```
Comment (caouecs) [inline sl/pagination.php]: Hello,
no need

**Comment by caouecs** [comment]
Thanks, can you do same work for Laravel5 in laravel5 branch ?

**Comment by extra90** [comment]
Yes, I will do ASAP.

On Mon, Feb 9, 2015 at 5:44 PM, Fred Delrieu <notifications@github.com>
wrote:

> Thanks, can you do same work for Laravel5 in laravel5 branch ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/caouecs/Laravel4-lang/pull/343#issuecomment-73542483>.
>



-- 
Lep Pozdrav,
Mitja Rogl
mitja.rogl@gmail.com

**Comment by extra90** [comment]
I just updated.

**Comment by caouecs** [comment]
We need validation.timezone : https://github.com/caouecs/Laravel4-lang/issues/259

**Comment by extra90** [comment]
Updated.

**Comment by caouecs** [comment]
thanks

**Pull request merged**